posts/2023-12-18_admiral_1_0/admiral_1_0.qmd

```diff
@@ -0,0 +1,289 @@
+
+```{r, fig.align = 'center', echo=FALSE}
+knitr::include_graphics("check.png")
+```
+
+
+The argument has a deprecated tag in the function documentation and will issue a warning to users. There was quite a bit of renaming of arguments for `1.0.0` so there are quite of few of these tags in our documentation. In subsequent releases, these arguments will be removed. Please see the [changelog](https://pharmaverse.github.io/admiral/news/index.html#admiral-100) if you would like to explore other functions that had arguments renamed. The issues are linked to each rename so you can follow along with the discussions!
+
+## Bug Fixes
+
+We love fixing bugs and take them incredibly seriously - especially when identified by members from the community.  
+
+If you find a pesky bug, please fill out a [Bug Report](https://github.com/pharmaverse/admiral/issues/new/choose) on our Issues Tab. 
+
+Each bug fixed by our development team is documented in our [changelog](https://pharmaverse.github.io/admiral/news/index.html#updates-of-existing-functions-1-0-0) with the Issue linked. 
+
+```{r, fig.align = 'center', echo=FALSE}
+knitr::include_graphics("issue_tag.png")
+```
+
+
+For example, if you click through the issue for `derive_extreme_event()` that identified a problem where the `condition` was ignored if the `mode` argument was used, you can see the Bug Report along with a reproducible example. You can also see the Pull Request for the exact code changes that are addressing this bug! Way cool!
+
+
+# Commitment to Stability
+
+admiral was born out of a conversation between Thomas Neitmann and Michael Rimler almost three years ago. Check out this [history blog post](https://pharmaverse.github.io/blog/posts/2023-10-10_pharmaverse_story/pharmaverse_story.html) for some of the play by play. During that time, we have worked at a furious pace to produce functions that can do almost all derivations needed for common ADaMs. However, we have noticed that some functions are very similar to others and have sought to consolidate to reduce the number of functions. Unfortunately, this creates a lot of deprecation and frustration for users.
+
+Enter Superseded!!
+
+Now with the release of `1.0.0` we are going to _superseded_ functions going forward. This is the process where when we introduce new functions to replace other functions, we will no longer deprecate the replaced functions.  Now we will supersede them, i.e. we will keep the function in the codebase for a long time, but recommend the use of the new function. If the function were to be removed from the codebase this will be done many years from the time it is superseded.  Please note, that superseded functions that have bugs identified will be fixed.   
```

Comment:
```suggestion
Now with the release of `1.0.0` we are going to _supersede_ functions going forward. This is the process where when we introduce new functions to replace other functions, we will no longer deprecate the replaced functions.  Now we will supersede them, i.e. we will keep the function in the codebase for a long time, but recommend the use of the new function. If the function were to be removed from the codebase this will be done many years from the time it is superseded.  Please note, that superseded functions that have bugs identified will be fixed.   
```

Reply:
In the last sentence, it's worth hammering the point home even further: "Please note that although superseded functions are not in scope for active development, any bugs identified within them will nevertheless be fixed".

Reply:
updated